cmd/createtree/main_test.go

```diff
@@ -157,10 +157,9 @@ func TestRun(t *testing.T) {
 		tree, err := createTree(ctx, test.opts)
 		switch hasErr := err != nil; {
 		case hasErr != test.wantErr:
-			t.Errorf("%v: createTree() returned err = '%v', wantErr = %v", test.desc, err, test.wantErr)
-			return
+			t.Fatalf("%v: createTree() returned err = '%v', wantErr = %v", test.desc, err, test.wantErr)
```

Comment:
Could we do `t.Errorf(...) / continue`? I think that was the original intent.

I probably had the test wrapped on a `func(){ ... }()` due to defers, but forgot to change the `returns` to `continues` after removing it.